.jenkins/job_beam_PostCommit_Java_RunnableOnService_Gearpump.groovy

```diff
@@ -33,8 +33,8 @@ mavenJob('beam_PostCommit_Java_RunnableOnService_Gearpump') {
 
   // Sets that this is a PostCommit job.
   // 0 5 31 2 * will run on Feb 31 (i.e. never) according to job properties.
-  // This job triggers only on SCM.
-  common_job_properties.setPostCommit(delegate, '0 5 31 2 *')
+  // This job triggers only on SCM, precommit is disabled as well.
```

Comment:
At first glance I thought it wouldn't work against the appropriate branch, but I see now that it should be fine. It'll fail if people not on the gearpump-runner branch run it, but that seems fine to me.